packages/mask/background/utils/injectScript.ts

```diff
@@ -0,0 +1,75 @@
+import { memoize } from 'lodash-es'
+
+export const injectedScriptURL = '/js/injected-script.js'
+export const maskSDK_URL = '/js/mask-sdk.js'
+const contentScriptURL = '/generated__content__script.html'
+
+export async function evaluateContentScript(tabId: number | undefined, frameId?: number) {
```

Comment:
Can load any plugin at the runtime by using this code?

Reply:
I strongly object to this idea although this is technically possible in Manifest V2 when we don't have our codebase secured.